src/System.Reflection/tests/AssemblyTests.cs

```diff
@@ -287,7 +287,7 @@ public void Load_Invalid()
         }
 
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Assembly.LoadFile() not supported on UapAot")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Assembly.LoadFile() not supported on UWP")]
```

Comment:
Are these SkipOnTargetFrameworks correct?  I'm wondering if the target moniker was wrong and the attributes should just be deleted.

Reply:
LoadFile is blocked on both uap and uapaot

Reply:
LoadFile is blocked on uap because of uap tried to artificially simulate the key limitations of uapaot. These quirks do not make sense anymore with uapaot gone.

I expect that the whole `TargetFrameworkMonikers.Uap` and its quirks will go away too, but we may wait until we have the detailed plan for UWP figured out with that.